src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphSubmissionDAOSpec.scala

```diff
@@ -30,13 +30,13 @@ class GraphSubmissionDAOSpec extends FlatSpec with Matchers with OrientDbTestFix
 //      dao.save(workspace.namespace,workspace.name,testData.submission1,txn)
       assertResult(Some(testData.submission1)) {
         dao.get(workspace.namespace,workspace.name,testData.submission1.submissionId,txn) }
-      assertResult(Seq(testData.submission1, testData.submission2)) {
+      assertResult(Seq(testData.submissionTerminateTest, testData.submission1, testData.submission2)) {
         dao.list(workspace.namespace,workspace.name,txn).toSeq
       }
       assert(dao.delete(workspace.namespace,workspace.name,testData.submission1.submissionId,txn))
       assertResult(None) {
         dao.get(workspace.namespace,workspace.name,testData.submission1.submissionId,txn) }
-      assertResult(1) {
+      assertResult(2) {
```

Comment:
This madness is because both GraphSubmissionDAOSpec and SubmissionMonitorSpec use the same list of submissions in different ways. IMO we should break them out into separate workspaces / test databases so they're not coupled.